servicemap/rest_client.py

```diff
@@ -22,30 +33,17 @@ def list_helsinki_schools_and_kindergartens(self, filter_list=None):
             1257,  # General upper secondary education
             1097,  # Basic education
         ]
-
-        filter_params = (
-            self.__convert_to_string_param(filter_list) if filter_list else {}
-        )
-
+        payload = {
+            **filters,
+            # 91 = Helsinki - https://www.hel.fi/palvelukarttaws/rest/v4/arealcity/
+            "arealcity": "91",
+            "ontologytree": "+".join([str(node) for node in service_nodes]),
+        }
```

Comment:
How about
```python
payload = filters or {}
payload["arealcity"] = "91"  # Helsinki - https://www.hel.fi/palvelukarttaws/rest/v4/arealcity/
payload["ontologytree"] = "+".join([str(node) for node in service_nodes])
```
or
```python
payload = filters or {}
payload.update({
    "arealcity": "91",  # Helsinki - https://www.hel.fi/palvelukarttaws/rest/v4/arealcity/
    "ontologytree": "+".join([str(node) for node in service_nodes]),
})
```
or
```python
payload = {
    **(filters or {}),
    # 91 = Helsinki - https://www.hel.fi/palvelukarttaws/rest/v4/arealcity/
    "arealcity": "91",
    "ontologytree": "+".join([str(node) for node in service_nodes]),
}
```
to make sure `TypeError: 'NoneType' object is not a mapping` wouldn't happen it `filters` parameter is not overriden?

I think the first one would be supported by "Flat is better than nested." from Zen of Python. I'd vote for the first one, to me it reads clearest, it is most easy to read IMHO but YMMV.